### PR TITLE
MCS-841: Database Refactor

### PR DESCRIPTION
**Pull request by brianpippin**
This handles ingesting to specific collections for each eval.  Script to update database to move collections out on their own.

Also handles MCS-1101 to remove old legacy folder ingestion code.

### CODE REVIEW

File: scripts/_0_5_0_migrate_collections.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+def migrate_collections(mongoDB):
```
Comment (deanwetherby) [inline scripts/_0_5_0_migrate_collections.py:1]: Can this function be split into smaller testable functions?

Comment (brianpippin) [reply scripts/_0_5_0_migrate_collections.py:30]: Added some smaller functions to improve readability.

File: mcs_scene_ingest.py [hunk 2]
```diff
@@ -18,10 +18,31 @@
...
+        }
+    )
+
+    if mapping is None:
+        eval_number = float(re.sub("[^0-9.]", "", eval_name))
+        collection_name = "eval_" + (str(eval_number)).replace(".", "_") + "_scenes"
+        collection.insert_one({"name": eval_name, "collection": collection_name})
+        return collection_name
+    else:
+        return mapping["collection"]
```
Comment (ThomasSchellenbergNextCentury) [inline mcs_scene_ingest.py:43]: You've got two `get_scene_collection` functions that are very similar. Should the function be extracted to a separate file, and then imported into both mcs_history_ingest and mcs_scene_ingest?

File: mcs_scene_ingest.py [hunk 3]
```diff
@@ -50,20 +72,16 @@ def automated_scene_ingest_file(
...
 
-    if eval_name is None:
-        scene["eval"] = scene["debug"]["evaluation"]
-        eval_name = scene["debug"]["evaluation"]
-    else:
-        scene["eval"] = eval_name
+    scene["eval"] = scene["debug"]["evaluation"]
 
-    scene["evalNumber"] = int(re.sub("[^0-9]", "", scene["eval"]))
+    scene["evalNumber"] = float(re.sub("[^0-9.]", "", scene["eval"]))
```
Comment (ThomasSchellenbergNextCentury) [inline mcs_scene_ingest.py:84]: You're making the evalNumber into a float in a few places. I understand that's important to support Eval 3.5, but won't that also make "Eval 4" into "Eval 4.0"? Will that be OK?

**Pull request merged**